pkg/storage/storage.go

```diff
@@ -13,7 +13,8 @@ var (
 	RequestTimeout = 10 * time.Second
 
 	// DatastoreRetries defines how many retries are attempted when updating the Pool
-	DatastoreRetries = 100
+	DatastoreRetries  = 100
+	PodRefreshRetries = 5
```

Comment:
this might not be needed if we end up using `wait.PollImmediate` or `wait.ExponentialBackoff` .